src/coreclr/jit/eeinterface.cpp

```diff
@@ -596,6 +596,29 @@ const char* Compiler::eeGetShortClassName(CORINFO_CLASS_HANDLE clsHnd)
     return printer.GetBuffer();
 }
 
+//------------------------------------------------------------------------
+// eeGetClassAssemblyName:
+//   Get the assembly name of a type.
+//   If missing information (in SPMI), then return a placeholder string.
+//
+// Parameters:
+//   clsHnd - the handle of the class
+//   buffer - a buffer to use for scratch space, or null pointer to allocate a new string.
```

Comment:
This takes buffer and bufferSize arguments, but they are never used.

Copy&paste?

Reply:
It's a leftover after 0f0c7ef. Resloved.